client/app/scripts/liveblog-edit/freetype.service.js

```diff
@@ -152,9 +154,10 @@ function(angular) {
                             return all;
                         }
                     });
-                    return '<li ng-repeat="' + iteratorName + ' in ' + SCOPE_FREETYPEDATA + '.' + vector + '">' +
-                            repeater + '<freetype-collection-remove index="$index" vector="' + SCOPE_FREETYPEDATA + '.' + vector + '"/>' +
-                            '</li><li><freetype-collection-add vector="' + SCOPE_FREETYPEDATA + '.' + vector + '"/></li>';
+                    collection = SCOPE_FREETYPEDATA + '.' + vector;
+                    return '<li ng-repeat="' + iteratorName + ' in ' + collection + '">' +
```

Comment:
Surely there must a better way to do that. Why should template strings be injected in a service?